Lesson17/Lesson17/gokce/script.js

```diff
@@ -0,0 +1,217 @@
+// 1. Write a loop that prints out the numbers from 1 to 10.
+console.log("Exercise 1");
+for(let i=1; i<11; i++){
+console.log(i);
+}
+
+// 2. Write a loop that prints out the odd numbers from 1 to 20.
+console.log("Exercise 2");
+for(let i=1; i<=18; i+=2){
+    console.log(i);
+}
+// 3. Write a loop that prints out the even numbers from 1 to 20.
+// 2, 4, 6...
+
+console.log("Exercise 3");
+for(let i=2; i<=18; i+=2){
```

Comment:
Here again, it should be 'i<=20' as exercise says "prints out the even numbers from 1 to 20."